### ISSUE TITLE
New style blocks significant parts of the page depending on screen size

### ISSUE THREAD

**Issue by earth-tone** [rtfd/readthedocs.org#4753]
## Details

At some screen sizes, significant parts of the page are blocked by the new styles. Depending on how large my browser window is, between 2-3 of the opening paragraphs are obscured. It looks like you're using Bootstrap (or at least the sizes at which content changes appearance are the same sizes as Bootstrap); if you are, then these are the styles governed by the "large" and "x-large" selectors.

Screenshots:

Small and medium (up to 990px) displays correctly: 
![image](https://user-images.githubusercontent.com/12517389/46899588-e0e4ad80-ce62-11e8-9618-f9007bdb1116.png)

Large (990-1200px) cuts off the 3 opening paragraphs: 
![image](https://user-images.githubusercontent.com/12517389/46899599-02459980-ce63-11e8-897f-45c055e80cfc.png)

Extra large (over 1200px) Cuts off 2 opening paragraphs: ![image](https://user-images.githubusercontent.com/12517389/46899591-e80bbb80-ce62-11e8-80d6-fe027f051795.png)

## Expected Result

I expect all of the content to display

## Actual Result

At screen sizes over 990px, portions of the content are not visible

**Comment by earth-tone** [comment]
It's just the new CSS theme. The text is just obscured by the header. 

Here's the URL: https://redditclient.readthedocs.io/en/latest/

My username is earth-tone. The project is MkDocs. I don't know how to build it myself locally, sorry.

**Comment by stsewd** [comment]
@earth-tone I think this is something happening with the latest version of mkdocs (or some update, as you say this was working before. You can install a previous version using a `requirements.txt` file  https://docs.readthedocs.io/en/latest/guides/specifying-dependencies.html

Closing as this is a problem with mkdocs, rtd don't touch or modify that theme.

**Comment by earth-tone** [comment]
This has nothing to do with mkdocs. This is about CSS. Are you retarded?

**Comment by earth-tone** [comment]
I've opened a new issue, to let people know the real issue, which is that your employees are hiding these problems to cover their own asses.

**Comment by davidfischer** [comment]
I agree with @humitos and @stsewd. Build your project outside RTD and see for yourself.
